src/CFamily.UnitTests/CFamily.UnitTests.csproj

```diff
@@ -30,5 +30,26 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+
+  <ItemGroup Label="HACK to make the C++ rule loader tests work">
+    <!-- We need all of the json files in the lib subfolder to be copied under the test assembly folder.
+         The files are marked as content files in the Integration project, but since this project was
+         converted to be an SDK-style project the files are being deleted when before the tests are run,
+         causing them to fail. 
+    -->
+    <Content Include="..\Integration.Vsix\lib\**\*">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Link>lib\%(Filename)%(Extension)</Link>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Moved from the Integration.Vsix.UnitTests project and tweak to use wildcards to include all `.txt` and `.json` files rather than listing them explicitly.